Player.java

```diff
@@ -1,5 +1,7 @@
+package vjezba_xo;
+
 public interface Player {
     String getSymbol();
-    int[] makeMove();
+    int[] makeMove(Player player1, Player player2, String[][] board);
```

Comment:
U redu je proslijediti "board", ali playere ne bas, vidim da ih nidje i ne koristis, pa mi nije najjasnije zasto si ovo uradio. 
Ima smisla reci: odigraj potez, evo ovo je tabla, tako da je to skroz regularno rjesenje. Uvijek se treba truditi da se proslijedi sto manje nepotrebnih podataka, tako da je bolje proslijediti samo tablu, jer to je sve sto bi igracu trebalo da bude potrebno da izabere svoj potez. 
Mozda nije losa ideja ni da napravis novi tip "Board" koji ce da apstrahuje podatke o tabli, i da ima metode tipa: placeSymbolOn(String symbol, String coordinate); isFieldAvailable(String coordinate) i slicno. Na ovaj nacin odvajas logiku koja nema veze sa igracem na njeno mijesto. 
Istrazi pojam "tight coupling", uvijek se trudi da objekat bude sto nezavisniji, i da ima sto manje referenci na druge objetke, osim kada to ima smisla

Reply:
Iskren da budem ne znam ni sam što sam ih proslijedio, vjerovatno sam mislio da će mi trebati za provjeru ko pravi trenutni potez ali se ispostavilo da mi ne trebaju.
Kreirao sam klasu Board, i potrudio se da stvari koje imaju veze sa board-om rješavam u toj klasi.
Za ubuduće ću znati da ne treba da "vezujem" klase na ovaj način nego da ih razdvojim što je više moguće, jer jedna izmjena u glavnoj klasi povlači puno izmjena na mjesta gdje se koristi.